### ISSUE TITLE
Plugin activation failed message

### ISSUE THREAD

**Issue by Pejic88** [thomasgriffin/TGM-Plugin-Activation#276]
I have a problem when installing external plugins from zip files. There is a message that plugin activation failed and message after said that plugin is successfully installed and activated.
![1](https://cloud.githubusercontent.com/assets/2436669/6635173/ee893284-c962-11e4-85ff-4ac44c92f572.png)

**Comment by GaryJones** [comment]
Is it just the two ACF extensions? How about if ACF is already installed and active? I wonder if the extensions check to see if ACF is already active before potentially disabling the extensions if it's not.

**Comment by Pejic88** [comment]
Yeah, it happens with that two ACF extension. Since this is my first use of this TGM, I can't tell you is that happening with some other zipped plugins. As you can see from the screenshot, I have put ACF to be installed and activated first to enable extensions to be installed without obstacles.

**Comment by GaryJones** [comment]
If you manually install and activate ACF first, then trigger the TGM-activated install of the extensions, do you still get the error?

**Comment by Pejic88** [comment]
![3](https://cloud.githubusercontent.com/assets/2436669/6654062/fd430b4c-caac-11e4-97a7-921c20f9e683.PNG)

Still the same errors

**Comment by shivapoudel** [comment]
@GaryJones why so many commits gap between master and develop branch. Would be better if master branch is managed with updates xD

**Comment by Pejic88** [comment]
The problem is in this config option "is_automatic". When it's set to true, I got the errors, but when it's set to false, there is no errors, but user have to manually activate plugins after install.

**Comment by GaryJones** [comment]
@shivapoudel We're planning to do a v3, so we'll review the v2 branch and do one more release on that.

@Pejic88 Your screenshot still shows ACF being installed via TGM.

**Comment by Pejic88** [comment]
@GaryJones please take a closer look, there is no ACF on that screen, only ACF extensions.

**Comment by GaryJones** [comment]
My bad :-)

@elliotcondon - any insights as to why ACF extensions activate (or not) differently?

**Comment by elliotcondon** [comment]
Hey guys

I'm not sure, but the screenshot shows some empty errors. Can you check the AJAX responses / console log to see what PHP issues are going on?

**Comment by Pejic88** [comment]
I checked it right now and there is no errors at console log neither in AJAX responses.

**Comment by elliotcondon** [comment]
Hi @Pejic88 

Can you confirm that PHP errors are showing? Have you enabled WP_DEBUG mode?

**Comment by Pejic88** [comment]
Hi @elliotcondon 

Sry, I don't fully understand you in the previous comment. I was looking only at browser network inspection and console. Now I have enabled WP_DEBUG mode and this is what I got.
![screencapture-localhost-wp-wp-admin-themes-php](https://cloud.githubusercontent.com/assets/2436669/6662492/613925d6-cbbc-11e4-863a-dc76b895871d.png)

**Comment by GaryJones** [comment]
@Pejic88 Are you running master or develop branch code of TGMPA?

**Comment by Pejic88** [comment]
@GaryJones - master branch

**Comment by shivapoudel** [comment]
@Pejic88 use develop branch xD

**Comment by Pejic88** [comment]
@shivapoudel why?

**Comment by shivapoudel** [comment]
@Pejic88 While using the develop branch I haven't got any console log error after running with WP_DEBUG enables xD

**Comment by Pejic88** [comment]
@GaryJones do you recommend use of develop branch or not?

**Comment by GaryJones** [comment]
For debugging, it's worth trying it with the develop branch in case we've already fixed the problem, though I don't think anything related to the problem you've shown here was changed in the develop branch.

**Comment by GaryJones** [comment]
@Pejic88 Did you try the develop branch? We've updated a bit since March (though nothing aimed specifically at fixing this Issue).

**Comment by jrfnl** [comment]
@Pejic88 We'll be releasing v2.5 any day now. Please test this to see if the problem has been solved or still exists. If solved, please close this issue.

If the bug *does* still exist, please provide us with a copy of the relevant parts of your TGMPA config function and links to the problem plugins if they are public or provide us with a copy of the files if they are not.

**Comment by weavertheme** [comment]
Sorry, I missed this thread, but posted the solution in a new thread.
